documentation/modules/exploit/linux/http/nagios_xi.md

```diff
@@ -0,0 +1,70 @@
+## Vulnerable Application
+	Nagios XI 5.2.6 - 5.4.12 Chained Remote Root RCE
+
+	This exploit combines many different vulnerabilities in Nagios XI to 
+  gain remote root access to the affected host. The process is:
+
+  1. Reset the database user to root.
+  2. Exploit SQL injection to extract api keys.
+  3. Use api key to add administrative user.
+  4. Authenticate to application using newly added user.
+  5. Exploit command injection and sudo misconfiguration 
+     to get remote root shell.
+  6. Remove added admin user, and reset database user.
+ 
+  See [our blog post](http://blog.redactedsec.net/exploits/2018/04/26/nagios.html) for more information
```

Comment:
The module documentation is the perfect place to list the software installers which you linked in the PR:

[Sample OVA for Nagios 5.4.10](https://assets.nagios.com/downloads/nagiosxi/5/ovf/nagiosxi-5.4.10-64.ova)